repos.bzl

```diff
@@ -1234,3 +1242,18 @@ def go_repositories():
         sum = "h1:SvFZT6jyqRaOeXpc5h/JSfZenJ2O330aBsf7JfSUXmQ=",
         version = "v0.0.0-20190308202827-9d24e82272b4",
     )
+
+def configure(minimum_bazel_version = None, rbe_name = "rbe_default", go_version = None, nogo = None, go_modules = go_repositories):
+    if minimum_bazel_version:  # Allow an additional downstream constraint
+        versions.check(minimum_bazel_version = minimum_bazel_version)
+    versions.check(minimum_bazel_version = "0.29.1")  # Minimum rules for this repo
+    rbe_autoconfig(name = "rbe_default")
+    protobuf_deps()  # No options
+
+    go_rules_dependencies()  # No options
+    go_register_toolchains(go_version = None, nogo = None)
+
+    gazelle_dependencies()  # TODO(fejta): go_sdk and go_repository_default_cache
+
+    if go_modules:
+        go_modules()
```

Comment:
I don't see `go_modules` set anywhere in this PR - what/how is this intended to be used?

Reply:
Wait, I've been really stupid and not just looked at the default value for this var 🙃